src/frontend/src/lib/components/send/Send.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	import { modalStore } from '$lib/stores/modal.store';
+	import { modalSend } from '$lib/derived/modal.derived';
+	import SendButton from '$lib/components/send/SendButton.svelte';
+	import SendModal from '$lib/components/send/SendModal.svelte';
+
+	const modalId = Symbol();
+</script>
+
+<SendButton on:click={() => modalStore.openSend(modalId)} />
+
+{#if $modalSend}
```

Comment:
shouldn't you check the modalId here?

Reply:
right! very sorry! I forgot it! included in the last commit